docker/docker-compose.yml

```diff
@@ -13,9 +17,9 @@ services:
     working_dir: /etc/nginx/sites-enabled
     volumes:
       - ./nginx/sites-enabled:/etc/nginx/sites-enabled:ro
-      - installdirectory/data/shared/sites:/data/shared/sites
+      - /Users/frankie/x-com/data/shared/sites:/data/shared/sites
```

Comment:
This shouldn't be hardcode, the install.sh script replaces installdirectory. Should work on Mac in the latest version